src/glb-director/packaging/glb-director.service

```diff
@@ -4,7 +4,8 @@ After=network.target network-online.target
 Wants=network-online.target
 
 [Service]
-Type=simple
+Type=notify
+NotifyAccess=main
```

Comment:
Might be worth a comment about what `main` is here if it's anything useful/might be something else in some situations.

Reply:
:+1: Added a note about it - it was actually the default when `Type=notify` is set but felt it was better to explicitly set it given that:
> If NotifyAccess= is missing or set to none, it will be forcibly set to main.